frontend/freebase2wikidata.js

```diff
@@ -318,6 +367,23 @@ $(function() {
       });
     })();
 
+     // Add create Primary Sources nav list button
+    (function createPSNavList() {
```

Comment:
Fixed. Now an empty navigation menu loads on start and builds up as soon as new
statements and claims get parsed.